Berksfile

```diff
@@ -0,0 +1,7 @@
+source 'https://supermarket.chef.io'
+
+solver :ruby, :required
+
```

Comment:
You should add `apt` here to make this work better on our workstations. But not in metadata since it's not technically a requirement. We normally pulls in base which already pulls that so we don't see this problem.

Reply:
Ah good point!